Resources/Prototypes/Entities/Objects/Weapons/Melee/spear.yml

```
@@ -1,27 +1,28 @@
-- type: entity
+﻿- type: entity
   name: spear
   parent: BaseItem
   id: Spear
   components:
   - type: Sprite
     sprite: Objects/Weapons/Melee/spear.rsi
     state: spear
-
   - type: MeleeWeapon
     range: 1.5
     arcwidth: 0
     arc: spear
-
   - type: Item
     size: 24
     sprite: Objects/Weapons/Melee/spear.rsi
     prefix: inhand
-
   - type: Construction
     graph: spear
     node: spear
-
   - type: ItemCooldown
+  - type: MeleeChemicalInjector
+  - type: SolutionContainer
+    maxVol: 5
+    caps: AddTo, RemoveFrom, FitsInDispenser
```

This probably shouldn't fit in a dispenser
```suggestion
    caps: AddTo, RemoveFrom
```